MAINTAINERS.yml

```diff
@@ -328,6 +327,28 @@ Bluetooth:
   tests:
     - bluetooth
 
+Bluetooth HCI:
+  status: maintained
+  maintainers:
+    - jhedberg
+    - jori-nordic
+  collaborators:
+    - hermabe
+    - alwa-nordic
+    - Thalley
+    - sjanc
+    - theob-pro
```

Comment:
collaborator list is imported from "Bluetooth Host", which is where the drivers were before.